tests/test_file_storage.py

```diff
@@ -450,8 +461,10 @@ def test_seek_file(file_storage, file_path, mode, location, whence, expected_siz
 @pytest.mark.parametrize("provider", [(FileStorageProvider.GCS)])
```

Comment:
@gaya3-zipstack by using it as a fixture, you avoid the need to parameterize it and pass to your tests

Reply:
Like I mentioned, I have  a use case where I need to support arguments to this function which er are trying to keep as fixture. I could not find a way to do that.